src/tokens.ts

```diff
@@ -142,13 +143,215 @@ export interface IDebugger {
    *
    * @param source The source object containing the path to the file.
    */
-  getSource(source: DebugProtocol.Source): Promise<IDebugger.ISource>;
+  getSource(source: DebugProtocol.Source): Promise<IDebugger.Source>;
 }
 
 /**
  * A namespace for visual debugger types.
  */
 export namespace IDebugger {
+  /**
+   * A namespace for UI interface definitions.
+   */
+  export namespace UI {
```

Comment:
Wondering whether this should be UI specific.

Could `Model` also be an adequate name?

Reply:
Yeah, I think that makes sense, too.